cmd/earthly/secret_cmds.go

```diff
@@ -168,16 +169,15 @@ func (app *earthlyApp) actionSecretsGetV2(cliCtx *cli.Context) error {
 		return err
 	}
 
-	secrets, err := cloudClient.ListSecrets(cliCtx.Context, path)
+	secret, err := cloudClient.GetUserOrProjectSecret(cliCtx.Context, path)
 	if err != nil {
+		if errors.Is(err, cloud.ErrNotFound) {
+			return errors.New("no secret found for that path")
```

Comment:
should we mention the path that was used in the error?

Reply:
seems a bit overkill, malformed paths should generally be caught higher up, e.g. parts will append a `/` prefix when none are given by the user.